src/mpid/ch4/src/ch4_init.h

```diff
@@ -455,7 +455,7 @@ MPL_STATIC_INLINE_PREFIX int MPID_Get_universe_size(int *universe_size)
         *universe_size = pvalue->data.uint32;
         PMIX_VALUE_RELEASE(pvalue);
     }
-#elif USE_PMI2_API
+#elif defined USE_PMI2_API
```

Comment:
Can you add parenthesis just for consistency with other instances in the code?

Reply:
Done.